director/settings.py

```diff
@@ -35,12 +34,12 @@ def __init__(self, home_path=None, config_path=None):
         env = Env()
         env.read_env(env_path)
 
+        self.VERSION = env.str("DIRECTOR_VERSION", version)
```

Comment:
hi @ncrocfer,
It will allow user to customize the footer. For my case, I can pass GIT_TAG from my CI/CD pipeline and see it in the UI.
regards,

Reply:
Ok I see, indeed it seems to be a very good use case ! You can for instance reference `celery-director==x.y.z` in your own `requirements.txt` file, then display your hash commit or git tag in the footer.

But I'm not sure the `DIRECTOR_VERSION ` is a good name for that (because the real director version doesn't change, so it brings confusion). I will check what is done by other similar tools for this kind of needs and I will answer you.